spec/requests/api/users_spec.rb

```diff
@@ -510,16 +510,7 @@
         end
 
         context '正しいユーザーデータが指定された場合' do
-          context 'emailがすでに存在するユーザのemailの場合' do
-            let(:params) { { email: user.email } }
-
-            it '422が返って、エラーメッセージを返すこと' do
-              expect(subject).to have_http_status :unprocessable_entity
-              expect(subject.parsed_body).to have_key('errors')
-            end
-          end
-
-          context 'emailのユーザが存在しない場合' do
+          context 'emailののユーザが存在しない場合' do
```

Comment:
typo
```suggestion
          context 'emailのユーザが存在しない場合' do
```

※ これも差分を見ているとき付けるタイプの typo ですね